CHANGES.md

```diff
@@ -1,3 +1,27 @@
+## edget-19.1.4
+
+* Controller
+  * Added timeouts support! Configurable in the service profiles for each route
+  * Added an experimental CNI plugin to avoid requiring the NET_ADMIN capability
+    when injecting proxies (more details at https://linkerd.io/2/cni)
+  * Added more improvements to the API for `ListPods` (thanks @alenkacz!)
+
+* Web UI
+  * Grayed-out the tap icon for requests from sources that are not meshed
+
+* CLI
+  * Added the `--proto` flag to `linkerd profile` to output a service profile
+    based on a Protobuf spec file -- Retryable methods can be declared in the Protobuf
+    file using the `@Retryable` annotation
```

Comment:
I don't think this use of the `@Retryable` annotation is implemented

Reply:
oh ok, I'll remove that :+1: